### PR TITLE
special_symbols_added

### PR DESCRIPTION
**Pull request by anushka1309paul**
The special symbols such as " , ", " @", " { ", " } ", " # ", " * " etc. are now added when a txt file is uploaded to the editor.

### CODE REVIEW

File: src/pages/Editor/containers/editContext.js [hunk 1]
```diff
@@ -182,13 +182,13 @@ const EditContextProvider = props => {
 
   function convertToPlain(rtf) {
     rtf = rtf.replace(/\\par[d]?/g, "");
-    rtf = rtf.replace(/\{\*?\\[^{}]+}|[{}]|\\\n?[A-Za-z]+\n?(?:-?\d+)?[ ]?/g, "");
+    rtf = rtf.replace(/\{\*?\\[^{}]+}|\\\n?[A-Za-z]+\n?(?:-?\d+)?[ ]?/g, "");
     // rtf = rtf.replace(/\n/ig, " ");
-    rtf = rtf.replace(/\\/gi, "");
-    rtf = rtf.replace(/\*/gi, "");
+    // rtf = rtf.replace(/\\/gi, "");
```
Comment (anushbhatia) [inline src/pages/Editor/containers/editContext.js:187]: don't add these in comments remove them. if they are of no use.

Comment (anushka1309paul) [reply src/pages/Editor/containers/editContext.js]: All right!

Comment (anushka1309paul) [reply src/pages/Editor/containers/editContext.js]: I have committed the necessary changes. Please go through the same and merge the pull request.

Comment (anushka1309paul) [reply src/pages/Editor/containers/editContext.js]: I have committed the necessary changes. Please go through the same and merge the pull request.

**Comment by anushbhatia** [comment]
squash commit otherwise other's changes will be reflected in yours.

**Pull request merged**